src/index.ts

```diff
@@ -77,7 +77,7 @@ const resso = <Obj extends Record<string, unknown>>(obj: Obj): Store<Obj> => {
         return useSyncExternalStore(
           state[key].subscribe,
           state[key].getSnapshot,
-          state[key].getSnapshot
```

Comment:
this doesn't seem right, third parameter is for server-side snapshot

Reply:
https://react.dev/reference/react/useSyncExternalStore#adding-support-for-server-rendering